config/locales/user_mailer/en.yml

```diff
@@ -29,10 +29,12 @@ en:
         want to delete your account?
       subject: Delete your account
     account_reset_request:
+      header: Wait 24 hours to delete your account
       help: ''
-      intro: You’ve requested to delete your login.gov account. <br><br><strong>Your
-        request will be processed in 24 hours.</strong> <br><br> If you don’t want
-        to delete your account, <u>%{cancel_account_reset}</u>.
+      intro: You’ve requested to delete your login.gov account. For security reasons,
```

Comment:
How would you feel about taking the opportunity to clean this up by breaking it into different sections and using a paragraph tag for each in the template to avoid tags in the translation?

Reply:
Sure! I'll do that

Reply:
@jmhooper Left the `<strong/>` tags in there but got rid of the `<br/>` tags, does that work? Or should I get rid of those as well?

Reply:
Yeah, totally. Getting rid of the paragraph tags is what I meant